BUFR_TableD_en_15.csv

```diff
@@ -293,6 +293,64 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,004025,Time period or displacement,,,,Operational
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,014017,Instantaneous long-wave radiation,,,,Operational
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,014018,Instantaneous short-wave radiation,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301150,WIGOS identifier,,,79,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,201129,Change data width,Add one bit to width of following descriptor,,,Operational
```

Comment:
sequence name differs from issue sumary: 

"Ocean glider trajectory profile sequence" vs  "Sequence for reporting observations from a single glider trajectory profile"

Reply:
I changed the name to be consistent with naming of the other sequences but we can use that originally proposed.

Reply:
Ok. I'll update the summary to reflect the branch